server/views/logged-in-logout-poc.ejs

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>SAM Solution | Logged-in/Logout POC</title>
```

Comment:
Rename all the POCs (titles and file names) by decent names for a production environment.